client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -57,17 +57,23 @@ class EditorMediaModalDetailItem extends Component {
 
 		const mimePrefix = MediaUtils.getMimePrefix( item );
 
-		if ( 'image' !== mimePrefix ) {
+		if ( ! includes( [ 'image', 'video' ], mimePrefix ) ) {
 			return null;
 		}
 
+		let editText = translate( 'Edit Image' );
+
+		if ( 'video' === mimePrefix ) {
+			editText = translate( 'Edit Thumbnail' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Thumbnail' )` *ES Score: 9.74*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/video-thumbnail-picker)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).